### PR TITLE
MM-18712 UI Automation: Large number of emoji reactions wrap into multiple lines

### PR DESCRIPTION
**Pull request by jganobsik**
#### Ticket Link

Resolves https://github.com/mattermost/mattermost-server/issues/12289

### CODE REVIEW

File: e2e/cypress/integration/emoji/wrap_emoji_spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+describe('Wrap emojis', () => {
+    before(() => {
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        cy.clearLocalStorage();
+    });
+
+    it('MM-18712 Large number of emoji reactions wrap into multiple lines', async () => {
+        //cy.get('#post_textbox').type(':cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: :cowboy_hat_face: {enter}')
+        cy.postMessage(':dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog: :dog:');
```
Comment (saturninoabril) [inline e2e/cypress/integration/emoji/wrap_emoji_spec.js]: I realized that the test requirement is a bit confusing (sorry for that).  It's not posting emojis but it's actually adding number of reactions to a post.

**Comment by saturninoabril** [comment]
Hello @jganobsik, just checking the status of this PR?

File: e2e/cypress/integration/emoji/wrap_emoji_spec.js (hunk 1 above)
Comment (jganobsik) [reply e2e/cypress/integration/emoji/wrap_emoji_spec.js]: Will address this weekend

**Comment by jganobsik** [comment]
> Hello @jganobsik, just checking the status of this PR?

Made several of the minor changes, just need to re-write test to reflect reactions instead of messages

**Comment by saturninoabril** [comment]
@jganobsik Thanks, please ping me once ready for review.

**Comment by saturninoabril** [comment]
@jganobsik Let me know if you need help.

**Comment by saturninoabril** [comment]
Closing due to inactivity.  Feel free to submit separate PR if still interested.

**Pull request closed without merging**